manifests/init.pp

```diff
@@ -80,7 +80,7 @@
   $sshd_ignoreuserknownhosts           = 'no',
   $sshd_ignorerhosts                   = 'yes',
   $manage_service                      = true,
-  $sshd_addressfamily                  = 'any',
+  $sshd_addressfamily                  = undef,
```

Comment:
use the `'USE_DEFAULTS"` pattern.

So if this is Solaris, it is `undef` and not Solaris it should be `'any'`